### PR TITLE
force ordering between initializing fromRepository and calling getImages

### PR DESCRIPTION
**Pull request by deads2k**
This requires the call to initialize the images happen before any call to get the images will be honored.

It also adds nil protection for the stop path of disruption checks.

### CODE REVIEW

File: test/extended/util/image/image.go [hunk 1]
```diff
@@ -38,14 +43,43 @@ func init() {
 		"registry.k8s.io/e2e-test-images/agnhost:2.43": 1,
 		"registry.k8s.io/e2e-test-images/nginx:1.15-4": 22,
 	}
+)
 
-	images = GetMappedImages(allowedImages, os.Getenv("KUBE_TEST_REPO"))
+func getImages() map[string]string {
+	initializationLock.RLock()
+	if !initialized {
+		fmt.Printf("Called getImages before initialization, starting wait.")
```
Comment (stbenjam) [inline test/extended/util/image/image.go:51]: Are we interested in figuring out where it's coming from? We could put [`runtime.Caller`](https://pkg.go.dev/runtime#Caller) info here

**Comment by deads2k** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-aws-ovn-upgrade 10

**Comment by stbenjam** [comment]
/lgtm

Release hold when payload passes

**Comment by deads2k** [comment]
found the underlying bug.  https://github.com/openshift/origin/pull/28123 contains this and all other fixes.

**Pull request closed without merging**